pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -340,54 +340,47 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	if plugin.loNetwork != nil {
 		// Loopback network deletion failure should not be fatal on teardown
 		if err := plugin.deleteFromNetwork(cniTimeoutCtx, plugin.loNetwork, name, namespace, id, netnsPath, nil); err != nil {
-			klog.Warningf("CNI failed to delete loopback network: %v", err)
+			klog.InfoS("CNI failed to delete loopback network", "err", err)
 		}
 	}
 
 	return plugin.deleteFromNetwork(cniTimeoutCtx, plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
-	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
-}
-
 func (plugin *cniNetworkPlugin) addToNetwork(ctx context.Context, network *cniNetwork, podName string, podNamespace string, podSandboxID kubecontainer.ContainerID, podNetnsPath string, annotations, options map[string]string) (cnitypes.Result, error) {
 	rt, err := plugin.buildCNIRuntimeConf(podName, podNamespace, podSandboxID, podNetnsPath, annotations, options)
 	if err != nil {
-		klog.Errorf("Error adding network when building cni runtime conf: %v", err)
+		klog.ErrorS(err, "Error adding network when building cni runtime conf")
 		return nil, err
 	}
 
-	pdesc := podDesc(podNamespace, podName, podSandboxID)
 	netConf, cniNet := network.NetworkConfig, network.CNIConfig
-	klog.V(4).Infof("Adding %s to network %s/%s netns %q", pdesc, netConf.Plugins[0].Network.Type, netConf.Name, podNetnsPath)
+	klog.V(4).InfoS("Adding pod to network", "pod", klog.KRef(podNamespace, podName), "podSandboxID", podSandboxID, "podNetnsPath", podNetnsPath, "networkType", netConf.Plugins[0].Network.Type, "networkName", netConf.Name)
 	res, err := cniNet.AddNetworkList(ctx, netConf, rt)
 	if err != nil {
-		klog.Errorf("Error adding %s to network %s/%s: %v", pdesc, netConf.Plugins[0].Network.Type, netConf.Name, err)
+		klog.ErrorS(err, "Error adding pod to network", "pod", klog.KRef(podNamespace, podName), "podSandboxID", podSandboxID, "podNetnsPath", podNetnsPath, "networkType", netConf.Plugins[0].Network.Type, "networkName", netConf.Name)
 		return nil, err
 	}
-	klog.V(4).Infof("Added %s to network %s: %v", pdesc, netConf.Name, res)
+	klog.V(4).InfoS("Added pod to network", "pod", klog.KRef(podNamespace, podName), "podSandboxID", podSandboxID, "networkName", netConf.Name, "res", res)
```

Comment:
What is "res" here? Can you check the type? It looks like response.
It that's the case, let's change the name to "response"

Reply:
res is the result of  executing a CNI plugin.
Can also be understood as a response.
I have fixed up it.